Documents/LearnJs/Constructor(prototype)/js/script.js

```diff
@@ -0,0 +1,95 @@
+//1
+function Car(make, model, year, color, price) {
+    this.make = make;
+    this.model = model;
+    this.year = year;
+    this.color = color;
+    this.price = price;
+}
+
+function Garage() {
+    this.cars = [];
+}
+
+Garage.prototype.addCar = function (make, model, year, color, price) {
+    const newCar = new Car(make, model, year, color, price);
+    this.cars.push(newCar);
+    return "New car was added: " + make + " " + model;
+}
+
+const myGarage = new Garage();
+
+
+function addCarToGarage() {
+    const make = document.getElementById("makeInput").value;
+    const model = document.getElementById("modelInput").value;
+    const year = parseInt(document.getElementById("yearInput").value);
+    const color = document.getElementById("colorInput").value;
+    const price = parseFloat(document.getElementById("priceInput").value);
+    myGarage.addCar(make, model, year, color, price);
+    document.querySelectorAll('input[type="text"], input[type="number"]').forEach(input => input.value = '');
```

Comment:
You can separate code by functionalities.  `document.querySelectorAll('input[type="text"], input[type="number"]').forEach(input => input.value = '');` can moved to separate function